soot-infoflow-android/src/soot/jimple/infoflow/android/callbacks/AbstractCallbackAnalyzer.java

```diff
@@ -646,7 +647,7 @@ protected boolean invokesInflate(InvokeExpr inv) {
 			if (curClass.getName().equals("android.app.Fragment"))
 				return true;
 			if (curClass.declaresMethod("android.view.View inflate(int,android.view.ViewGroup,boolean)"))
-				return false;
+				return true;
```

Comment:
Das kann raus, wenn man den Check darüber generalisiert, irgendeine Android Layout-Klasse zu nehmen. Dein aktueller Check akzeptiert auch eine Methode ``inflate()`` in User Code, der gar nichts mit Android zu tun hat.

Reply:
ich habe den mal angepasst, android.view.LayoutInflater deklariert die Methode auch.